pycape/llms/llms.py

```diff
@@ -100,7 +101,15 @@ async def completions(
 
         data = crypto.envelope_encrypt(
             self.ctx.public_key,
-            {"request": {"prompt": prompt, "stream": stream}, "user_key": user_key},
+            {
+                "request": {
+                    "prompt": prompt,
+                    "stream": True,
```

Comment:
We are removing the param but forcing it to true here?

Reply:
Sorry, I think my commit message is wrong. We're removing it from allowing the user to set it in the sdk because non-stream isn't supported. i.e. we want to force the sdk to use streaming.

Reply:
gotcha, thanks!